src/main/java/seedu/superta/logic/commands/ViewStudentFeedbackCommand.java

```diff
@@ -22,7 +22,7 @@
 public class ViewStudentFeedbackCommand extends Command {
 
     public static final String COMMAND_WORD = "view";
-    public static final String MESSAGE_SUCCESS = "Student ID: %s, Feedback:\n";
+    public static final String MESSAGE_SUCCESS = "Student ID: %s\n" + "Feedback:\n";
```

Comment:
I have modified this message in my latest PR. Sorry for any inconvenience!

Reply:
no prob!